main.tf

```diff
@@ -2,7 +2,10 @@
 # Get ID of created Security Group
 ##################################
 locals {
+  description = var.create_group ? var.description : "Not used"
+  name        = var.create_group ? var.name : "Not used"
   this_sg_id = concat(
+    [var.security_group_id],
```

Comment:
`this_sg_id` will always contain the first value, which is going to be an existing security group or an empty string. Please fix and run other examples to confirm that this PR does not break existing functionality.

Reply:
Fixed